### ISSUE TITLE
Spotfire 11 is Required But Not Mentioned in Spotfire Mods Documentation

### ISSUE THREAD

**Issue by KenACollins** [TIBCOSoftware/spotfire-mods#66]
Nowhere in the online documentation for Spotfire Mods (https://tibcosoftware.github.io/spotfire-mods) does it say that Spotfire Mods is a feature of Spotfire 11. Under the prerequisites listed on the Getting Started page, it just says "A running instance of Spotfire" is required. It does not specify a minimum version. The About page does not specify Spotfire 11 either.

This is a huge gaping hole in the documentation, considering that many customers. ourselves included, are running Spotfire 10 (it took the consultants 8 months to migrate us from version 7.9 so I have no idea when we can expect to be on 11).

Bottom line: When you roll out new features, be cognizant that many customers are on older versions so be crystal clear the minimum version of Spotfire necessary to utilize the feature.

**Comment by hski-github** [comment]
Agree that it would be good to mention it. You need actually minimum 11.3. The first LTS version supporting Mods is 11.4. 

As a side note, Cognizant is currently migration our Spotfire infrastructure from 10.3 to 11.4. Maybe these folks can talk to your Cognizant support team to speed up your upgrade.

**Comment by ebrandin** [comment]
The best available overview of the Spotfire Mods API versions and how they relate to Spotfire versions is available on GitHub.

https://github.com/TIBCOSoftware/spotfire-mods#spotfire-versions-and-their-mod-api
